evennia/contrib/utils/git/git_integration.py

```diff
@@ -0,0 +1,171 @@
+from django.conf import settings
+from evennia import CmdSet, InterruptCommand
+from evennia.utils.utils import list_to_string
+from evennia.commands.default.muxcommand import MuxCommand
+from evennia.server.sessionhandler import SESSIONS
+
+import git
+import datetime
+
+class CmdGit(MuxCommand):
+    """
+    Pull the latest code from your repository or checkout a different branch.
+
+    Usage:
+        git status        - View an overview of your git repository.
+        git branch        - View available branches.
+        git checkout main - Checkout the main branch of your code.
+        git pull          - Pull the latest code from your current branch.
+
+    For updating evennia code, the same commands are available with 'git evennia':
+        git evennia status
+        git evennia branch
+        git evennia checkout <branch>
+        git evennia pull
+        
+    If there are any conflicts encountered, the command will abort. The command will reload your game after pulling new code automatically, but for changes involving persistent scripts etc, you may need to manually restart.
+    """
+
+    key = "@git"
+    aliases = ["@git evennia"]
```

Comment:
No need to hardcode the `@` prefix. You'll be able to use `@git` anyway if you prefer.